scripts/runTests.fish

```diff
@@ -136,6 +136,7 @@ set CT "$CT""250,runClusterTest1 dump_maskings - --dumpAgencyOnError true\n"
 set CT "$CT""250,runClusterTest1 dump_multiple - --dumpAgencyOnError true\n"
 set CT "$CT""750,runClusterTest1 http_server - --dumpAgencyOnError true\n"
 set CT "$CT""250,runClusterTest1 hot_backup -\n"
+set CT "$CT""500,runClusterTest1 permissions -\n"
```

Comment:
please don't put this to the cluster tests, they already take more time. Pls use single above instead.

Reply:
Yes, thanks for checking. I fucked it up